leap/leap_test.rb

```diff
@@ -15,26 +18,44 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert_equal true, Year.leap?(1996)
   end
 
   def test_non_leap_year
     skip
-    refute Year.leap?(1997), 'No, 1997 is not a leap year'
+    assert_equal false, Year.leap?(1997)
   end
 
   def test_non_leap_even_year
     skip
-    refute Year.leap?(1998), 'No, 1998 is not a leap year'
+    assert_equal false, Year.leap?(1998)
   end
 
   def test_century
     skip
-    refute Year.leap?(1900), 'No, 1900 is not a leap year'
+    assert_equal false, Year.leap?(1900)
   end
 
   def test_fourth_century
     skip
-    assert Year.leap?(2400), 'Yes, 2400 is a leap year'
+    assert_equal true, Year.leap?(2400)
+  end
+
+  def test_Y2K
```

Comment:
As per previous comment, this was the name as per the json test definition, I'll sort it out.